### PR TITLE
[QuestionAnswering] Add new 2021-07-15-preview version 

### PR DESCRIPTION
**Pull request by rokulka**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Changelog
Add a changelog entry for this PR by answering the following questions:
  1. What's the purpose of the update?
      - [ ] new service onboarding
      - [ ] new API version
      - [ ] update existing version for new feature
      - [ ] update existing version to fix swagger quality issue in s360
      - [ ] Other, please clarify
  2. When are you targeting to deploy the new service/feature to public regions? Please provide the date or, if the date is not yet available, the month.
  3. When do you expect to publish the swagger? Please provide date or, the the date is not yet available, the month.
  4. If updating an existing version, please select the specific langauge SDKs and CLIs that must be refreshed after the swagger is published.
      - [ ] SDK of .NET (need service team to ensure code readiness)
      - [ ] SDK of Python
      - [ ] SDK of Java
      - [ ] SDK of Js
      - [ ] SDK of Go
      - [ ] PowerShell
      - [ ] CLI
      - [ ] Terraform
      - [ ] No refresh required for updates in this PR

### Contribution checklist:
- [ ] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of "no breaking changes"
- [ ] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [ ] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist 

> **Applicability**: :warning: 
>
> If your changes encompass only the following scenarios, you should SKIP this section, as these scenarios do not require ARM review.
> - Change to data plane APIs 
> - Adding new properties 
> - All removals

Otherwise your PR may be subject to ARM review requirements. Complete the following:
- [ ] Check this box if any of the following apply to the PR so that label “WaitForARMFeedback” will be added automatically to begin ARM API Review. Failure to comply may result in delays to the manifest.
  - Adding a new service
  - Adding new API(s)
  - Adding a new API version
    -[ ] To review changes efficiently, ensure you copy the existing version into the new directory structure for first commit and then push new changes, including version updates, in separate commits.

- [ ] Ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If any of the following scenarios apply to the PR, request approval from the Breaking Change Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 
- [ ] Removing API(s) in a stable version
- [ ] Removing properties in a stable version
- [ ] Removing API version(s) in a stable version
- [ ] Updating API in a stable or public preview version with Breaking Change Validation errors
- [ ] Updating API(s) in public preview over 1 year (refer to [Retirement of Previews](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37683/Retirement-of-Previews))

**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Addition details on the process and office hours are on the [Breaking change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

File: specification/cognitiveservices/data-plane/Language/preview/2021-05-01-preview/examples/questionanswering/SuccessfulQueryKnowledgebases.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
     "api-version": "2021-05-01-preview",
```
Comment (heaths) [inline specification/cognitiveservices/data-plane/Language/preview/2021-05-01-preview/examples/questionanswering/SuccessfulQueryKnowledgebases.json:6]: All these samples' api-versions need to be updated.

**Review by heaths — APPROVED** [review]
OpenAPI Hub shows no differences for questionanswering.json or questionanswering-authoring.json (apart from api-version, which it ignores for the differ):

* questionanswering.json: https://portal.azure-devex-tools.com/tools/diff/eyJzb3VyY2VVcmwiOiJodHRwczovL3Jhdy5naXRodWJ1c2VyY29udGVudC5jb20vQXp1cmUvYXp1cmUtcmVzdC1hcGktc3BlY3MvZGI0ODUzNjNkY2IwZjEzYjE4YjY3YzhmOTIyN2Y4ODg5OTdlYjc4MS9zcGVjaWZpY2F0aW9uL2NvZ25pdGl2ZXNlcnZpY2VzL2RhdGEtcGxhbmUvTGFuZ3VhZ2UvcHJldmlldy8yMDIxLTA1LTAxLXByZXZpZXcvcXVlc3Rpb25hbnN3ZXJpbmcuanNvbiIsInRhcmdldFVybCI6Imh0dHBzOi8vcmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbS9BenVyZS9henVyZS1yZXN0LWFwaS1zcGVjcy9kYjQ4NTM2M2RjYjBmMTNiMThiNjdjOGY5MjI3Zjg4ODk5N2ViNzgxL3NwZWNpZmljYXRpb24vY29nbml0aXZlc2VydmljZXMvZGF0YS1wbGFuZS9MYW5ndWFnZS9wcmV2aWV3LzIwMjEtMDctMTUtcHJldmlldy9xdWVzdGlvbmFuc3dlcmluZy5qc29uIn0=
* questionanswering-authoring.json: I can't actually tell here, but we didn't ship SDK support for authoring so it should be fine.

**Comment by heaths** [comment]
To note, the one diff URL compared files from different repos and branches, but then OpenAPI Hub gave me an error for a different file across both that they had to be the same repo and branch: https://portal.azure-devex-tools.com/tools/diff/eyJzb3VyY2VVcmwiOiJodHRwczovL3Jhdy5naXRodWJ1c2VyY29udGVudC5jb20vQXp1cmUvYXp1cmUtcmVzdC1hcGktc3BlY3MvZmVhdHVyZS9jb2duaXRpdmVsYW5ndWFnZS9xdWVzdGlvbmFuc3dlcmluZy1wcmV2aWV3LjEvc3BlY2lmaWNhdGlvbi9jb2duaXRpdmVzZXJ2aWNlcy9kYXRhLXBsYW5lL0xhbmd1YWdlL3ByZXZpZXcvMjAyMS0wNS0wMS1wcmV2aWV3L3F1ZXN0aW9uYW5zd2VyaW5nLWF1dGhvcmluZy5qc29uIiwidGFyZ2V0VXJsIjoiaHR0cHM6Ly9yYXcuZ2l0aHVidXNlcmNvbnRlbnQuY29tL3Jva3Vsa2EvYXp1cmUtcmVzdC1hcGktc3BlY3Mvcm9rdWxrYS92ZXJzaW9uL3NwZWNpZmljYXRpb24vY29nbml0aXZlc2VydmljZXMvZGF0YS1wbGFuZS9MYW5ndWFnZS9wcmV2aWV3LzIwMjEtMDctMTUtcHJldmlldy9xdWVzdGlvbmFuc3dlcmluZy1hdXRob3JpbmcuanNvbiJ9

/cc @akning-ms on the error and inconsistency here.

**Comment by ChongTang** [comment]
This looks good to me. I had a similar version change in my PR. But I will update more and merge it after this one merged.

File: specification/cognitiveservices/data-plane/Language/preview/2021-05-01-preview/examples/questionanswering/SuccessfulQueryKnowledgebases.json (hunk 1 above)
Comment (rokulka) [reply specification/cognitiveservices/data-plane/Language/preview/2021-05-01-preview/examples/questionanswering/SuccessfulQueryKnowledgebases.json:6]: Thanks for pointing it out, will do in the next PR.

**Review by anuchandy — APPROVED** [review]

**Pull request merged**